src/cloud/project/user-admin.md

```diff
@@ -176,8 +164,118 @@ Add an environment-level user:
 
 {%include cloud/note-prevent-site-availability-issues.md%}
 
+## Update account security settings
+
+After you add a user to a Cloud project, ask the user to review their account settings and add the following security configuration as needed:
+
+-  Enable two-factor authentication (TFA)
+
+   Magento recommends adding two-factor authentication to all accounts to meet security and compliance standards. Some projects will soon require two-factor authentication for all accounts that use SSH to connect to {{site.data.var.ece}} projects.
+
+-  Enable SSH keys
+
+   Users that require access to {{site.data.var.ece}} source code repositories and infrastructure must enable SSH keys on their account. See [Enable SSH keys][].
+
+-  Create an API token
+
+   You can generate an API token on your account that can be used for secure machine access to an environment through an SSH connection. You can use the API token to enable login workflows for automated processes.
+
+   On projects with the MFA enforcement feature enabled, you must use the API token to authenticate SSH access requests from automated accounts. to bypass authentication workflows which require 2FA.
+
+### Enable 2FA
+
+{{site.data.var.ece}} supports 2FA using any of the following applications:
+
+-  [Google Authenticator (Android/iPhone/BlackBerry)][]
+-  [Authy (Android/iPhone)][]
+-  [Authenticator (Windows Phone)][]
+-  [FreeOTP (Android)][]
+-  [GAuth Authenticator (Firefox OS, desktop, others)][]
+
+You can find instructions for setting up the authenticator app and enabling 2FA on the _Account settings_ page for your {{site.data.var.ece}} user account.
+
+{:.procedure}
+To enable 2FA on your {{site.data.var.ece}} user account:
+
+1. Log in to the [{{site.data.var.ece}} user account][{{site.data.var.ece}} account].
+
+1. On the Cloud projects page, click the **Account settings** tab.
+
+   ![Cloud projects page]({{ site.baseurl }}/common/images/cloud/cloud-account-settings-tab.png){:width="550px"}
+
+1. On the Account settings tab, expand the **Security** section. Then, click **Set up application**.
+
+   ![Cloud Security settings]({{ site.baseurl }}/common/images/cloud/
+   cloud_account_settings-tfa-setup-app.png){:width="550px"}
+
+1. Use the linked instructions to install one of the approved authentication applications on your mobile device, if needed.
+
+   ![Cloud Security settings]({{ site.baseurl }}/common/images/cloud/
+   cloud_account_settings-tfa-setup-app-options.png){:width="550px"}
```

Comment:
```suggestion
   ![Cloud Security settings]({{ site.baseurl }}/common/images/cloud/cloud_account_settings-tfa-setup-app-options.png){:width="550px"}
```

Reply:
Thanks for the suggestions-- I didn't see them before building and pushing from local. Build running now 🤞